config/site_settings.yml

```diff
@@ -580,7 +580,7 @@ files:
     client: true
     shadowed_by_global: true
   external_system_avatars_url:
-    default: "https://avatars.discourse.org/letter/{first_letter}/{color}/{size}.png"
+    default: "//{hostname}/letter_avatar_proxy/letter/{first_letter}/{color}/{size}.png"
```

Comment:
It looks like the class methods to get the path prefix, if there is one, are available: https://github.com/discourse/discourse/blob/master/lib/discourse.rb#L184-L211